Gemfile

```diff
@@ -27,7 +27,7 @@ group :rubygems do
 end
 
 group :sss do
-  gem 'aws-sdk-v1'
+  gem 'aws-sdk', '>= 2.0.18.pre', '< 2.0.38'
```

Comment:
When I remember correctly bundler was complaining otherwise that multiple versions of the same gem are used.

All AWS SDKv2 services are using this range https://github.com/travis-ci/dpl/blob/master/Gemfile#L39